templates/repo/release_tag_header.tmpl

```diff
@@ -0,0 +1,26 @@
+{{$canReadReleases := $.Permission.CanRead $.UnitTypeReleases}}
+{{$canReadCode := $.Permission.CanRead $.UnitTypeCode}}
+<div class="release-tag-header gt-df gt-ac">
+	<div class="release-tag-header-left gt-df gt-ac gt-f1">
+		{{if $canReadReleases}}
+			<h2 class="ui compact small menu header small-menu-items">
+				<a class="{{if .PageIsReleaseList}}active {{end}}item" href="{{.RepoLink}}/releases">{{.locale.Tr "repo.release.releases"}}</a>
+				{{if $canReadCode}}
+					<a class="{{if .PageIsTagList}}active {{end}}item" href="{{.RepoLink}}/tags">{{.locale.Tr "repo.release.tags"}}</a>
+				{{end}}
+			</h2>
+			{{if .EnableFeed}}
+				<a class="rss-icon" href="{{.RepoLink}}/{{if .PageIsTagList}}tags{{else}}releases{{end}}.rss" data-tooltip-content="{{.locale.Tr "rss_feed"}}">{{svg "octicon-rss" 18}}</a>
+			{{end}}
+		{{else if $canReadCode}}
+			{{template "repo/sub_menu" .}}
+		{{end}}
```

Comment:
Still have the question, why the "full width" sub_menu is put in `release-tag-header-left`?

Can the sub_menu be put in outside of these "left/right"?

Reply:
What exactly makes the menu "full width"? For all I know, it can render in any available space and is responsive.

Reply:
The "sub_menu" is this:

<details>

![image](https://github.com/go-gitea/gitea/assets/2114189/a5be2050-43cf-46a1-857d-580870faf648)

</details>

It is full-width, so it's strange to put it inside the `xxx-left`.

I won't say "block" but I strongly prefer to leave a comment for it: that's it used for, when it would be shown, and how to test it.

Reply:
I also know the context & background, I will propose a change and write some comments, to avoid some uncommon pages broken one day.

Reply:
Done from my side.